components/bpmn/bpmn-rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java

```diff
@@ -161,6 +181,13 @@ public ResponseHolder taskVariationOverTime(@PathParam("assignee") String assign
                     assignee + "'.");
         }
 
+        String taskAssignee ;
+        if ( tenantDomain == DOMAIN_OF_SUPER_TENANT ) {
```

Comment:
The string comparison is wrong. Is this intentional? seems to be a bug in the code